### PR TITLE
[DNM] mimic: mds: curate priority of perf counters sent to mgr

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/26991

### CODE REVIEW

**Comment by smithfarm** [comment]
```
In file included from /home/jenkins-build/build/workspace/ceph-pull-requests/src/mds/MDSRank.h:30:0,
                 from /home/jenkins-build/build/workspace/ceph-pull-requests/src/mds/MDSDaemon.h:30,
                 from /home/jenkins-build/build/workspace/ceph-pull-requests/src/mds/MDSDaemon.cc:41:
/home/jenkins-build/build/workspace/ceph-pull-requests/src/mds/SessionMap.h: In member function 'void Session::set_load_avg_decay_rate(double)':
/home/jenkins-build/build/workspace/ceph-pull-requests/src/mds/SessionMap.h:209:33: error: no matching function for call to 'DecayCounter::DecayCounter(double&)'
     load_avg = DecayCounter(rate);
                                 ^
```

File: src/mds/SessionMap.h [hunk 1]
```diff
@@ -30,6 +30,7 @@ using std::set;
 
 class CInode;
 struct MDRequestImpl;
+class DecayCounter;
```
Comment (smithfarm) [inline src/mds/SessionMap.h]: @batrick What to put here for mimic?

**Comment by smithfarm** [comment]
@batrick This is the problem, right?

```
$ git branch --contains bc7cb5798593a29a636cb53412b73856d4e3a093 | grep mimic
$
```

**Comment by batrick** [comment]
I'm going to ask @vshankar to do this backport due to the number of conflicts.

**Comment by vshankar** [comment]
ACK -- I'll do the backport

**Comment by smithfarm** [comment]
see https://github.com/ceph/ceph/pull/24467

**Pull request closed without merging**